src/sagas/ses.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { MMKV } from 'react-native-mmkv';
+import RNRestart from 'react-native-restart';
+import {
+  call,
+  fork,
+  take,
+  all,
+} from 'redux-saga/effects';
+import { checkForFeatureFlag } from './helpers';
+import { SES_FEATURE_TOGGLE, SHOULD_ENABLE_SES_STORAGE_KEY } from '../constants';
+
+const storage = new MMKV();
+
+function verifySesEnabled() {
+  return Object.isFrozen([].prototype);
```

Comment:
suggestion: Explain the reason for this unusual syntax on this function's docstrings.

Reply:
Refactored the method and added a better jsDoc in ea15c73ebde51c0a19ec914d2b9519f581086820